src/components/TagContainer.ts

```diff
@@ -0,0 +1,237 @@
+import { Component, createElement } from "react";
+
+import { BootstrapStyle, Tag } from "./Tag";
+import { parseStyle } from "../utils/ContainerUtils";
+
+interface WrapperProps {
+    class?: string;
+    mxObject: mendix.lib.MxObject;
+    readOnly: boolean;
+    style?: string;
+}
+
+export interface TagContainerProps extends WrapperProps {
+    afterCreateMicroflow: string;
+    allowDrag: boolean;
+    tagStyle: BootstrapStyle;
+    editable: "default" | "never";
+    enableSuggestions: boolean;
+    enableCreate: boolean;
+    inputPlaceholder: string;
+    onChangeMicroflow: string;
+    lazyLoad: boolean;
+    tagAttribute: string;
+    tagConstraint: string;
+    tagEntity: string;
+    tagLimit: number;
+    tagLimitMessage: string;
+}
+
+export interface TagState {
+    alertMessage?: string;
+    suggestions: string[];
+    tagList: string[];
+    newTag: string;
+}
+
+class TagContainer extends Component<TagContainerProps, TagState> {
```

Comment:
`export default  class` (not at the end of the file

Reply:
done